boot.js

```diff
@@ -189,7 +188,14 @@ function root (s, opts, done) {
   }
 }
 
-inherits(Boot, EE)
+Boot.prototype = Object.create(EventEmitter.prototype, {
+  constructor: {
+    value: Boot,
+    enumerable: false,
+    writable: true,
+    configurable: true
+  }
+})
```

Comment:
I think the equivalent is different:

https://github.com/nodejs/node/blob/9cc7327979a95a76662a5f044118bad1009fa5a8/lib/internal/streams/duplex.js#L41